modules/local/nf-core-modified/mafft.nf

```diff
@@ -27,12 +27,8 @@ process MAFFT {
     prefix=\$(basename "${fasta}" .fa)
 
     mafft \\
-        --thread ${task.cpus} \\
-        --localpair \\
-        --maxiterate 1000 \\
-        --anysymbol \\
-        ${fasta} \\
-        > \$prefix-mafft.fa
+        --thread -1 \\
```

Comment:
hmm we sure about this?

Reply:
I think it's safter than what I was doing. As is, I was specifying a fixed number of threads, but mafft should be able to autoscale for memory at the memory intensive stages in a sensible way with this flag.